### PR TITLE
Test with default gemfile

### PR DESCRIPTION
**Pull request by franzliedke**
I couldn't get the tests to run locally until adding the `responders` gem (which is used in tests).

So I also added testing with this Gemfile to Travis to ensure problems like that cannot go unnoticed.

### CODE REVIEW

**Comment by franzliedke** [comment]
I just pushed these two commits separately again, to show that the second commit indeed fixes the issue:

- [Build for first commit](https://travis-ci.org/jgraichen/decorate-responder/builds/474385705) - Fails as expected, but interestingly not for the Ruby-2.2/default-gemfile combination.
- [Build for second commit](https://travis-ci.org/jgraichen/decorate-responder/builds/474373435)

**Comment by jgraichen** [comment]
We're only testing for Rails 4.2+ at all which is the version since `responders` is required. How about adding it as an official dependency (and kind of officially dropping support for Rails < 4.2)?

**Comment by franzliedke** [comment]
I didn't know that `responders` is part of a typical Rails app. In that case, your suggestion sounds good.

**Comment by jgraichen** [comment]
It is not but responders have been removed from Rails itself with 4.2. Since that the `responders` gems is needed for having responders in your app and we can add it as an requirement.

**Comment by franzliedke** [comment]
Oh.... I misread your first comment. I thought you were suggestion to add Rails as an "official dependency", but apparently you meant responders. Which is great. :+1:

So you want me to add this to the `gemspec` instead of the `Gemfile`?

**Comment by jgraichen** [comment]
Jep. The `responders` dependency could be removed from the CI gemfiles too.

**Comment by franzliedke** [comment]
Done.

File: test/test_helper.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 require 'action_dispatch'
 
+require 'responders'
```
Comment (franzliedke) [inline test/test_helper.rb:11]: Not sure why this is necessary (there's a `Bundler.require` call a few lines above), but I needed this to make `bundle exec rake test` work locally.

Comment (jgraichen) [reply test/test_helper.rb:11]: `Bundle.require` will only load dependencies declared in the Gemfile not in gemspecs, and you just removed all these lines. ;)

Comment (franzliedke) [reply test/test_helper.rb:11]: But the Gemfile includes the gemspec?

Comment (jgraichen) [reply test/test_helper.rb:11]: But only like `gem 'decorate-responder', path: '.'`. It still will *not* load gems runtime dependencies.

**Pull request merged**